joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/X2Cpg.scala

```diff
@@ -173,6 +173,7 @@ trait X2CpgFrontend[T <: X2CpgConfig[?]] {
   /** Create CPG according to given configuration, printing errors to the console if they occur. The CPG is closed and
     * not returned.
     */
+  @throws[RuntimeException]("if createCpg throws any Throwable")
```

Comment:
Well, either `RuntimeException`, or `Throwable`, but not both, right?

Reply:
Yeah, right. Will change that.